commands/js.js

```diff
@@ -111,9 +111,9 @@ const js_call = {
     desc: 'Call a method on a contract',
     builder: (yargs) => yargs
         .option('accountId', {
-            desc: 'Unique identifier for the account that will be used to sign this call',
+            desc: 'Unique identifier for the account that will be used to sign this call. Will use the contract-id if not specified',
             type: 'string',
-            required: true,
+            default: null,
         })
```

Comment:
@volovyk-s is there some clever/cleaner way to do what I'm doing with setting the `default: null`, but maybe tell yargs that we want the value of `options.contractId`?

Reply:
@ChaoticTempest if `accountId` is not necessary - we should delete this option.

Is it possible that `accountId` can affect the result somehow?

`accountId` option deletion is a breaking change. It means that we should keep it, but log a warning that it's deprecated. `default: 'string'` line should be deleted and `accountId` should not be used anywhere.

Reply:
accountId is still required in some cases here since user might want to specify a different signer than the contractId. This is just a convenience addition so that people don't need to specify it twice, so no breaking change

Reply:
Oh, I got it wrong. I was sure that we are talking about a view call.

Reply:
hmm, yeah thinking about it again, let's just be explicit here and stay consistent with regular near-cli call command